cloud_testing/kotlin_poc/src/test/kotlin/ftl/TestArtifact.kt

```diff
@@ -0,0 +1,93 @@
+package ftl
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.io.File
+import java.util.concurrent.TimeUnit
+import kotlin.math.pow
+import xctest.Bash
+
+abstract class TestArtifact {
```

Comment:
since this is almost a direct copy and paste from the other project, I think we'd want to refactor this to a common `test_fixture` project that's referenced by both `kotlin_poc` and `xctest_parser`

otherwise we'd have to maintain the code in two different places.